File: app/src/main/java/com/codeliver128/shaadiinvitation/MyApplication.java
package com.codeliver128.shaadiinvitation;

import android.app.Application;
import android.content.Intent;
import com.codeliver128.shaadiinvitation.services.SoundService;

public class MyApplication extends Application {

    @Override
    public void onCreate() {
        super.onCreate();
        try {
//start service and play music
            startService(new Intent(this, SoundService.class));

        } catch (Exception ex) {
        }
    }

    @Override
    public void onTerminate() {
        stopService(new Intent(this, SoundService.class));
        super.onTerminate();
    }
}

File: app/src/main/java/com/invitation/MyApplication.kt
package com.invitation

import android.app.Application
import androidx.multidex.MultiDexApplication
import com.google.firebase.FirebaseApp

class MyApplication : MultiDexApplication() {
    override fun onCreate() {
        super.onCreate()
            FirebaseApp.initializeApp(this)
        try {
//            startService(Intent(this, SoundService::class.java))
        } catch (ex: Exception) {
        }
    }

    override fun onTerminate() {
//        stopService(Intent(this, SoundService::class.java))
        super.onTerminate()
    }
}